portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,9 @@
   <body>
     <div id="content">
       <h1>Elijah's Portfolio</h1>
-      <p>I was born and raised in Chicago Illinois, and I currently study Computer Science
-          at the University of Illinois At Chicago.
-      </p>
```

Comment:
Did you intend to remove this entirely or move it under Bio?

Reply:
I removed it entirely, I wanted to rewrite it, and leave this pull-request exclusively for the skeleton of the page.